code/camera/camera.cpp

```diff
@@ -54,6 +54,9 @@ APPLY_TO_FOV_T(*, mul)
 APPLY_TO_FOV_T(+, add)
 APPLY_TO_FOV_T(-, sub)
 
+// Used to set the default value for in-game options
+static float fov_default = DEFAULT_FOV;
```

Comment:
If we keep this one, it should be also be `const` or `constexpr`.

Reply:
Done